### DISCUSSION TITLE
Feature Request: 64 bit raspberry pi version of cli

### DISCUSSION THREAD

**Discussion by ashish01** [duckdb/duckdb#5940] [Ideas]
Why do you want this feature?

Makes it convenient to use duckdb in another environment.

**Comment by Mause** [comment]
This is already available: https://github.com/duckdb/duckdb/releases/download/v0.6.1/duckdb_cli-linux-rpi.zip

**Reply by ashish01** [reply]
This one is a 32-bit ELF binary.

```
ashish@raspberrypi /tmp> wget https://github.com/duckdb/duckdb/releases/download/v0.6.1/duckdb_cli-linux-rpi.zip
ashish@raspberrypi /tmp> unzip duckdb_cli-linux-rpi.zip
ashish@raspberrypi /tmp> file duckdb
```

The result of above commands is

```
duckdb: ELF 32-bit LSB executable, ARM, EABI5 version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-armhf.so.3, for GNU/Linux 2.6.32, with debug_info, not stripped
```

**Comment by eitsupi** [comment]
@ashish01 Did you try CLI for linux aarch64? https://github.com/duckdb/duckdb/releases/download/v0.6.1/duckdb_cli-linux-aarch64.zip

**Reply by ashish01** [reply]
Indeed aarch64 binaries work just fine. Thanks!

**Reply by eitsupi** [reply]
That's good to know; given that the Raspberry Pi is already arm64, perhaps the website needs to be updated.......
